src/NzbDrone.Core/Movies/Movie.cs

```diff
@@ -107,45 +107,33 @@ public bool IsAvailable(int delay = 0)
 
             //This more complex sequence handles the delay
             DateTime minimumAvailabilityDate;
-            switch (MinimumAvailability)
+
+            if ((MinimumAvailability == MovieStatusType.TBA) || (MinimumAvailability == MovieStatusType.Announced))
```

Comment:
What's the point of us having both of these if they do the same thing? Nothing to fix here, but something to think on

Reply:
ya, i dont know about them --> they were just there, before I implemented the MinimumAvailability feature. At the time preDB was also a thing, so there is code for that too. If I understand correctly. PreDB is gone, so in theory it should be removed from the Availability stuff too. Can we just remove the TBA moviestatus type as well? the other ones make sense: 

Announced, InCinemas, Released..

I have no idea what the difference was supposed to be between TBA and Announced.

Reply:
I opened an issue to track cleaning this up, since another one of my PR's touched an area with Deleted MovieStatusType which is also likely unneeded. https://github.com/Radarr/Radarr/issues/5002